module_intros/density.CorrelationFunction-Grains.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>Change the comment for <code>cf.correlation</code> to <code># Get the value of the correlation function</code>; we're not grabbing raw bin counts.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>